recipes/json-c/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: json-c
+    version: 0.12
+
+source:
+    fn: json-c-0.12.tar.gz
+    url: https://s3.amazonaws.com/json-c_releases/releases/json-c-0.12.tar.gz
+    md5: 3ca4bbb881dfc4017e8021b5e0a8c491
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - autoconf
+        - automake
+        - libtool
+    run:
+        - libgcc
```

Comment:
This has saved my in `nco` and `cdo`, until we have our `libgcc` I guess I will risk it and leave this there.